src/sentry/integrations/opsgenie/client.py

```diff
@@ -58,11 +62,13 @@ def _get_issue_alert_payload(
         rules,
         event: Event | GroupEvent,
         group: Group | None,
+        priority: OpsgeniePriority | None = "P3",
```

Comment:
nit: seems like p3 is default priority. Want to store that in a const or something?

Reply:
done!